.github/workflows/self-hosted.yml

```diff
@@ -19,8 +19,17 @@ env:
 
 jobs:
   self-hosted:
-    runs-on: ubuntu-latest
```

Comment:
I believe this change is causing ci builds to fail for the 25.6.0 release:
https://github.com/getsentry/publish/actions/runs/15688973336/job/44199206790